dbms/src/Common/MyTime.h

```diff
@@ -114,7 +114,8 @@ struct MyTimeBase
 
     // Check validity of time under specified SQL_MODE.
     // May throw exception.
-    void check(bool allow_zero_in_date, bool allow_invalid_date) const;
+    // return false if time is invalid
+    bool check(bool allow_zero_in_date, bool allow_invalid_date) const;
```

Comment:
Maybe rename the function name to be `isValid`?

Reply:
And should fix the comment in line 116

Reply:
okk